pandapipes/converter/stanet/valve_pipe_component/create_valve_pipe.py

```diff
@@ -0,0 +1,94 @@
+from pandapipes.component_models.component_toolbox import add_new_component
+from pandapipes.pandapipes_net import pandapipesNet
+from pandapower.auxiliary import get_free_id, _preserve_dtypes
+from pandapipes.converter.stanet.valve_pipe_component.valve_pipe_component import ValvePipe
+
+
+def create_valve_pipe(net, from_junction, to_junction, length_km, diameter_m, opened=True,
+                      loss_coefficient=0, sections=1, std_type=None, k_mm=0.15e-3,
+                      max_vdot_m3_per_s=100, max_v_m_per_s=None, alpha_w_per_m2k=0., qext_w=0.,
+                      name=None, index=None, in_service=True, type="valve_pipe", **kwargs):
+    """
+    Creates a valve pipe element in net["valve_pipe"] from valve pipe parameters.
```

Comment:
A little description how this differs from "pipe" and "valve" would be helpful (also for the autodoc). Maybe also, why this is hidden in the Stanet converter and should not be used otherwise

Reply:
I extended the docstrings.